### DISCUSSION TITLE
Cosmetic Typo in `Your Github Copilot setup is incomplete.` banner

### DISCUSSION THREAD

**Discussion by patrickshiel** [community/community#19415] [Copilot Conversations]
Hi Github team, 

Just noticed a typo in the message for the `Github Copilot setup is incomplete.` banner message. 

**blow => below**

`Your Github Copilot setup is incomplete. Select your preferences blow to start using Github Copilot`

=> 

`Your Github Copilot setup is incomplete. Select your preferences below to start using Github Copilot`

<img width="1160" alt="Screenshot 2022-06-25 at 12 19 36" src="https://user-images.githubusercontent.com/4146313/175771324-d0443a88-ca9e-40eb-8682-56dc8237139b.png">

**Comment by melvincwng** [comment]
Hi @patrickshiel, thanks for highlighting this issue to Github Community. 

For bugs, you can report it to [Github Bug Report](https://support.github.com/contact/bug-report).

Hope this helps you and clarifies.

Thank you, and have a nice day 😊
